src/modules/Dropdown/Dropdown.jsx

```diff
@@ -505,6 +509,7 @@ export default {
           this.item && 'item',
           this.floating && 'floating',
           this.fluid && 'fluid',
+          this.$options.propsData.hasOwnProperty('pointing') && `pointing ${this.pointing}`,
```

Comment:
Is there a specific reason why you can't use `this.pointing`?

Reply:
I want to check if the prop is passed or no. If prop is passed but no value given, I assume dev wants to point to default position (top center). If value (in this case value is position of pointer) is passed, then use it instead. 

Since propsData contains only props that are passed [without defaults], I can check if user passed prop with empty value or with any value. Using `this.pointing` I cannot detect that [or can't imagine how could I do that :) ], because default value would be applied.